docs/protocol/networking/networks.md

```diff
@@ -0,0 +1,75 @@
+# Networks
+
+Information about the default Ceramic networks
+
+## Overview
+---
+
+Networks are collections of Ceramic [nodes](../nodes/index.md) that share specific configurations and communicate over dedicated [libp2p](https://libp2p.io/) pubsub topics. Networks are disjoint from one another; streamsthat exist on one network are **not** discoverable or usable on another.
+
+These pubsub topics are used to relay all messages for the defined networking sub protocols, including [Tip Gossip](tip-gossip.md) and [Tip Queries](tip-queries.md). 
+
+## All Networks
+---
+
+An overview of the various Ceramic networks available today:
+
+| Name | Network ID | Ceramic Pubsub Topic | Timestamp Authority | Type |
+| --- | --- | --- | --- | --- |
+| Mainnet | mainnet | /ceramic/mainnet | Ethereum Mainnet (EIP155:1) | Public |
+| Clay Testnet | testnet-clay | /ceramic/testnet-clay | Ethereum Gnosis Chain | Public |
+| Dev Unstable | dev-unstable | /ceramic/dev-unstable | Ethereum Gnosis Chain | Public |
+| Local | local | /ceramic/local-$(randomNumber) | Ethereum by Truffle Ganache | Private |
+| In-memory | inmemory |  | None | Private |
+
+!!!note
+    As each Ceramic network is decomposed into multiple pubsub topics for scalability, the pubsub topics will remain prefixed by the network identifier `/ceramic/<network>/<sep>` see [CIP-120](https://github.com/ceramicnetwork/CIP/pull/120/files)
```

Comment:
This note shouldn't sound so certain. We can mention that there is a proposal to break pubsub into multiple topics.